File: spigot/src/main/java/de/s42/spigot/SpigotMessageService.java
package de.s42.spigot;

import de.s42.core.MessageService;
import net.md_5.bungee.api.ChatMessageType;
import net.md_5.bungee.api.chat.TextComponent;
import org.bukkit.ChatColor;
import org.bukkit.entity.Player;

public class SpigotMessageService implements MessageService {

    @Override
    public void sendActionBar(Player player, String message) {
        String coloredMessage = ChatColor.translateAlternateColorCodes('&', message);
        player.spigot().sendMessage(ChatMessageType.ACTION_BAR, new TextComponent(coloredMessage));
    }
}

File: spigot/src/main/kotlin/tr/s42/spigot/SpigotMessageService.kt
package tr.s42.spigot

import tr.s42.core.MessageService
import net.md_5.bungee.api.ChatMessageType
import net.md_5.bungee.api.chat.TextComponent
import org.bukkit.ChatColor
import org.bukkit.entity.Player

class SpigotMessageService : MessageService {

    override fun sendActionBar(player: Player, message: String) {
        val coloredMessage = ChatColor.translateAlternateColorCodes('&', message)
        player.spigot().sendMessage(ChatMessageType.ACTION_BAR, TextComponent(coloredMessage))
    }
}